### PR TITLE
Refactor proposal: simpler variables

### PR DESCRIPTION
**Pull request by luixo**
I'm not sure that's the way, but I would like to hear your opinions

### CODE REVIEW

File: backend/src/db-utils.ts [hunk 1]
```diff
@@ -205,7 +205,9 @@ const queryStakingPoolAccountIds = async (): Promise<string[]> => {
       [
         `SELECT account_id as "accountId"
     FROM accounts
-    WHERE account_id LIKE '%${nearStakingPoolAccountSuffix}'`,
+    WHERE account_id LIKE '%${
+      config.accountIdSuffix.stakingPool[config.networkName]
+    }'`,
```
Comment (shelegdmitriy) [inline backend/src/db-utils.ts:210]: Will this query works normally in case of 'localhostnet'?

**Review by shelegdmitriy — COMMENTED** [review]
There is much better to follow up the code, I like it!

File: backend/src/db-utils.ts (hunk 1 above)
Comment (luixo) [reply backend/src/db-utils.ts:210]: The line will be `WHERE account_id LIKE '%undefined'` and will return zero rows.
Looks good enough to me :)

File: backend/src/config.ts [hunk 2]
```diff
@@ -1,134 +1,53 @@
...
-  variableName: string,
-  defaultValue: number
-): number =>
-  getEnvWithDefault(
-    variableName,
-    (v) => {
-      const num = parseInt(v || "");
-      return isNaN(num) ? undefined : num;
+/*
+To override a config parameter you should use a specific environment variable, following rules:
```
Comment (luixo) [inline backend/src/config.ts:6]: @frol 
How do you like new naming system for env variables?

File: render.yaml [hunk 3]
```diff
@@ -121,16 +126,16 @@ services:
...
           type: web
           name: wamp
           property: host
-      - key: NEAR_EXPLORER_WAMP_BACKEND_SECRET
+      - key: NEAR_EXPLORER_CONFIG__WAMP__BACKEND_SECRET
         fromService:
           type: web
           name: wamp
-          envVarKey: NEAR_EXPLORER_WAMP_BACKEND_SECRET
+          envVarKey: NEAR_EXPLORER_CONFIG__BACKEND_SECRET
```
Comment (frol) [inline render.yaml:138]: 1. Don't forget to update the secrets env names on Render
2. FYI, You will need to manually remove the unused variable names as Render does not remove those automatically

**Review by frol — APPROVED** [review]
It looks much cleaner! 🎉

**Review by shelegdmitriy — APPROVED** [review]

File: backend/src/config.ts [hunk 4]
```diff
@@ -1,134 +1,53 @@
...
+
+For example, to override `config.accountIdSuffix.stakingPool.localhostnet` you should use variable
+NEAR_EXPLORER_CONFIG__ACCOUNT_ID_SUFFIX__STAKING_POOL__LOCALHOSTNET
+*/
+export const config = merge(
+  {
+    archivalRpcUrl: "http://localhost:3030",
+    networkName: "localhostnet" as NetworkName,
+    accountIdSuffix: {
+      lockup: "lockup.near",
```
[the remark this answers is not in the document]
Comment (luixo) [reply backend/src/config.ts:20]: That's a great idea

File: render.yaml (hunk 3 above)
Comment (luixo) [reply render.yaml:138]: Sure.
I'll make another PR to migrate DB env variables to this system as well, because it requires a little more effort

**Comment by luixo** [comment]
It took quite a research to make both next build and jest work with common modules though it works now.
We'll probably migrate to swc + custom plugin for stitches friendly class names instead of babel later, but it'll take a PR in next.js team (I'll open an issue later).

**Pull request merged**